panel/panel.sublime-syntax

```diff
@@ -47,8 +47,9 @@ contexts:
 
   expect-filename:
     - include: pop-at-end
-    - match: ([^:]+)
-      scope: entity.name.filename.sublime_linter
+    - match: (.+):(?=\n)
```

Comment:
I'd lean towards keeping it a bit more specific, e.g.

```
((\S:)?[^:]+):
```

Reply:
Done